docs/integrations/orca.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Orca
+head:
+  - - meta
+    - name: title
+      content: Solana Cookbook | Integrating Orca in your App
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | Integrating Orca in your App
+  - - meta
+    - name: description
+      content: Orca is the easiest place to exchange cryptocurrency on the Solana blockchain. On Orca, you can exchange tokens more cheaply, quickly, and confidently.
+  - - meta
+    - name: og:description
+      content: Orca is the easiest place to exchange cryptocurrency on the Solana blockchain. On Orca, you can exchange tokens more cheaply, quickly, and confidently.
+  - - meta
+    - name: og:image
+      content: https://solanacookbook.com/cookbook-sharing-card.png
+  - - meta
+    - name: og:image:alt
+      content: Solana splash card
+  - - meta
+    - name: twitter:card
+      content: summary
+  - - meta
+    - name: twitter:site
+      content: "@solanacookbook"
+  - - meta
+    - name: twitter:image
+      content: "https://solanacookbook.com/cookbook-sharing-card.png"
+  - - meta
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+---
+
+# Orca
+
+Orca is a Dex, which helps to exchange cryptocurrencies on the Solana Blockchain as quickly, cheaply and confidently as possible.
+You can start interacting with orca program by using Orca Javascript SDK and the @solana/web3.js library.
+
+## Installing Dependencies
+
```

Comment:
Need some information about this section, just a little blurb. Also, why `decimal.js`?